### DISCUSSION TITLE
Adding a source URL label to Dockerfile

### DISCUSSION THREAD

**Discussion by frantathefranta** [karlomikus/bar-assistant#393] [Ideas]
Hi, would it be possible to add this to the Dockerfile?:
```
LABEL org.opencontainers.image.source="https://github.com/karlomikus/bar-assistant"
```

The reason for this change being that [renovatebot/renovate](https://github.com/renovatebot/renovate) cannot currently fetch the changelog, as the docker image name differs from the Github repo address. I didn't open a pull request as it seems like a very minor change but I can it's easier. Obviously it'd be nice to apply this to salt-rim as well.

**Comment by zhdenny** [comment]
Hey, what do you do with renovate in relation to Bar Assistant? Like how are these related? Just curious if this is something I'd like to use for other apps

**Reply by frantathefranta** [reply]
I run bar assistant in a Kubernetes cluster, which is directed by a [git repo](https://github.com/frantathefranta/home-ops), where renovate runs to keep everything up-to-date.

**Comment by frantathefranta** [comment]
Thanks @karlomikus, works great!
